move-spec-test/src/main.rs

```diff
@@ -26,7 +26,16 @@ pub struct Opts {
 
 fn main() -> anyhow::Result<()> {
     let opts = Opts::parse();
-    let package_path = opts.package_path.unwrap_or(PathBuf::from("."));
+
+    let package_path = if let Some(path) = opts.package_path {
```

Comment:
Just thinking: shouldn't we create a function in `move-mutator` or even `move-mutator-common` - something like - `check_cli_params` and then just use it?

Now we'd have the same code in spec-test, mutator etc - even this PR introduces exactly the same code in two crates 😄

Reply:
Done

Reply:
I almost ended up refactoring CLIOptions in both `move-mutator` and `move-spec-test` 😄 
But this new solution is a quick compromise